### PR TITLE
Add details about deprecated runtimes

### LINKED ISSUE #3591: Add clarification about only first deprecated runtime deployment in an org requires support action
### Issue Type

Inaccurate, misleading, or out-of-date information

### Links to Affected Docs

https://docs.astronomer.io/astro/upgrade-runtime#run-a-deprecated-astro-runtime-version

### The Issue or Context

We don't currently mention to users that they can create deployments with deprecated runtimes after at least one has been created in their organization.

### Fix or Proposed Change

Add clarification to section.

### Additional Notes

- https://astronomer.slack.com/archives/C01UA03TPV1/p1711663680271149

### Required Reviewers

_No response_

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3591

### CODE REVIEW

File: astro/upgrade-runtime.md [hunk 1]
```diff
@@ -135,9 +135,15 @@ When you choose a deprecated version of Astro Runtime, Astronomer recommends alw
 :::
 
 1. Contact your Astronomer account team and request the ability to run deprecated versions of Astro Runtime.
-2. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your deprecated Astro Runtime version using the `astroRuntimeVersions` property.
+2. Create a [Personal Access Token](https://cloud.astronomer.io/token)(PAT) to use for authentication to Astro.
+    :::note
```
Comment (jwitz) [inline astro/upgrade-runtime.md:139]:
```suggestion
 
       :::info
```

File: astro/upgrade-runtime.md [hunk 2]
```diff
@@ -135,9 +135,15 @@ When you choose a deprecated version of Astro Runtime, Astronomer recommends alw
...
 
-After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md).
+    You only need a PAT for the first Deployment in your Organization running a deprecated version of the Runtime.
+
+    :::
```
Comment (jwitz) [inline astro/upgrade-runtime.md:143]:
```suggestion
    :::
    
```

File: astro/upgrade-runtime.md [hunk 3]
```diff
@@ -135,9 +135,15 @@ When you choose a deprecated version of Astro Runtime, Astronomer recommends alw
 :::
 
 1. Contact your Astronomer account team and request the ability to run deprecated versions of Astro Runtime.
-2. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your deprecated Astro Runtime version using the `astroRuntimeVersions` property.
+2. Create a [Personal Access Token](https://cloud.astronomer.io/token)(PAT) to use for authentication to Astro.
+    :::note
 
-After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md).
+    You only need a PAT for the first Deployment in your Organization running a deprecated version of the Runtime.
```
Comment (jwitz) [inline astro/upgrade-runtime.md:141]:
```suggestion
    You only need a PAT to create the first Deployment in your Organization running a deprecated version of Astro Runtime. You can then create all subsequent Deployments using either a Workspace or Organization API token.
```
Couldn't hurt to clarify this here as well

File: astro/upgrade-runtime.md [hunk 4]
```diff
@@ -135,9 +135,15 @@ When you choose a deprecated version of Astro Runtime, Astronomer recommends alw
...
+
+    :::
+3. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your deprecated Astro Runtime version using the `astroRuntimeVersions` property.
```
Comment (jwitz) [inline astro/upgrade-runtime.md:144]:
```suggestion
3. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your PAT in the authorization header, and your deprecated Astro Runtime version using the `astroRuntimeVersions` property.
```

File: astro/upgrade-runtime.md [hunk 5]
```diff
@@ -135,9 +135,15 @@ When you choose a deprecated version of Astro Runtime, Astronomer recommends alw
...
+
+After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md). You can also create additional Deployments with deprecated Runtimes using a [Deployment API Token](deployment-api-tokens.md).
```
Comment (jwitz) [inline astro/upgrade-runtime.md:146]:
```suggestion
After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md). You can also create additional Deployments with deprecated Runtimes using the Astro API and either a Workspace or Organization API token.
```

File: astro/upgrade-runtime.md [hunk 6]
```diff
@@ -135,9 +135,17 @@ When you choose a deprecated version of Astro Runtime, Astronomer recommends alw
 :::
 
 1. Contact your Astronomer account team and request the ability to run deprecated versions of Astro Runtime.
-2. Create a Deployment using the [Astro API](https://docs.astronomer.io/api/platform-api-reference/deployment/create-deployment). In your request, specify your deprecated Astro Runtime version using the `astroRuntimeVersions` property.
+2. Create a [Personal Access Token](https://cloud.astronomer.io/token)(PAT) to use for authentication to Astro.
+ 
+       :::info
 
-After you create the Deployment, you can manage the Deployment using any interface, including the Astro CLI and [Astro UI](deployment-settings.md).
+    You only need a PAT to create the first Deployment in your Organization running a deprecated version of Astro Runtime. You can then create all subsequent Deployments using either a Workspace or Organization API token.
```
Comment (lzdanski) [inline astro/upgrade-runtime.md:142]:
```suggestion
    You only need a PAT to create the first Deployment in your Organization running a deprecated version of Astro Runtime. You can then create all subsequent Deployments using either a Workspace or Organization API token with permissions to create a Deployment.
```

**Pull request merged**